jbpm-test-util/src/main/java/org/jbpm/test/persistence/scripts/quartzmockentities/QrtzJobDetails.java

```diff
@@ -57,6 +57,6 @@
     private Boolean requestsRecovery;
 
     @Lob
-    @Column(name = "JOB_DATA")
+    @Column(name = "JOB_DATA", length = 5000)
```

Comment:
This is a per this PR's note:

Note: QrtzJobDetails class modified to set a minimum BLOB type - Otherwise, when enabling Hibernate auto-create flag to true, it will create a default value of 250 bytes for BLOB columns which is not enough in some scenarios.

Problem with this is when database creation is through sql scripts. If you remember, we had to remove the blob size out of the scripts, cause it failed. I was thinking about setting SQL script to this size but at the same time these scripts are taken from Quartz repository. 

What do you think?

Reply:
Cannot remember now what failed when blob size was in the scripts. But these mock entities are used to validate the database scripts so I think the scripts and these entities should be 1:1?

Reply:
These Quartz scripts are taken from official Quart repo, so initially there was a problem with them when the size of the blob was set to 2000. So Maciej, removed the size in a PR, however afterwards when upgrading jbpm to use a newer version of Quartz (2.x), then these sql scripts overwrote these changes and put this size/limit back again.

So to avoid this, I raised a PR but on Quartz, so we make sure these size/limit is removed from the sql script in Quartz itself:

https://github.com/quartz-scheduler/quartz/pull/559

Problem now... if we remove this size also from the entities to keep them in sync with the scripts, we have a different problem because by default, hibernates generates a 250 bytes length for a blob field which is not enough for us (it fails), so I had to manually set this limit to a higher value, after all these are just mock entities.

Question is: should we really remove the size of the Quartz script or just increase to a certain valid size for us? Apparently in the original quartz script, it's kind of saying that you can increase these limit:

https://github.com/quartz-scheduler/quartz/blob/master/quartz-core/src/main/resources/org/quartz/impl/jdbcjobstore/tables_db2.sql

What do you think?